.travis.yml

```diff
@@ -183,6 +199,14 @@ jobs:
         - make image-build-helm
         - make image-push-helm
 
+    # Build and deploy arm64 scorecard-proxy docker image
+    - <<: *deploy
+      name: Docker image for scorecard-proxy (arm64)
+      arch: arm64
+      script:
+        - make image-build-scorecard-proxy
+        - make image-push-scorecard-proxy
+
     # Build and deploy amd64 scorecard-proxy docker image
```

Comment:
The Travis changes were verified here: https://travis-ci.org/github/camilamacedo86/operator-sdk/builds/668911026?utm_source=github_status&utm_medium=notification (See that these jobs are just executed against master and we need to do a fork pr to check them).

<img width="1349" alt="Screenshot 2020-03-31 at 00 11 10" src="https://user-images.githubusercontent.com/7708031/77970505-3f664b80-72e4-11ea-837c-c507dcae4110.png">
<img width="1344" alt="Screenshot 2020-03-31 at 00 11 06" src="https://user-images.githubusercontent.com/7708031/77970508-40977880-72e4-11ea-8da9-f4fdaa431606.png">
<img width="1336" alt="Screenshot 2020-03-31 at 00 11 02" src="https://user-images.githubusercontent.com/7708031/77970510-41300f00-72e4-11ea-966a-58e6020bc321.png">

c/c @joelanford